apps/src/templates/courseOverview/ResourcesEditor.js

```diff
@@ -57,11 +78,14 @@ export default class ResourcesEditor extends Component {
   }
 
   render() {
-    const { resources } = this.state;
+    const { resources, errorString } = this.state;
+
+    // avoid showing multiple empty reosurces
+    const lastNonEmpty = _.findLastIndex(resources, ({type, link}) => link && type);
 
     return (
       <div>
-        {resources.map((resource, index) =>
+        {resources.slice(0, lastNonEmpty + 2).map((resource, index) =>
```

Comment:
Oh, and you keep the `resources` array at its max size and take advantage of the `.slice` behavior so this doesn't add an editor that lets you go over maxResources.

This all might be too clever for me, but if it ain't broke...